modules/person.rb

```diff
@@ -0,0 +1,31 @@
+require_relative './nameable'
+
+class Person < Nameable
```

Comment:
- [ ] As suggested earlier and according to the project requirement, kindly implement a method to create a new rental using the `Rentals` class.

 ```
def rent_book(book, date)
    Rental.new(date, book, self)
  end
```